core/ir/aarch64/instr_create_api.h

```diff
@@ -18494,4 +18494,82 @@
  */
 #define INSTR_CREATE_wfit(dc, Rt) instr_create_0dst_1src(dc, OP_wfit, Rt)
 
+/**
+ * Creates a LD64B instruction.
+ *
+ * This macro is used to encode the forms:
+ * \verbatim
+ *    LD64B <Xt>, [<Xn|SP> {, #0}]
+ * \endverbatim
+ * \param dc   The void * dcontext used to allocate memory for the #instr_t.
+ * \param Rt0  The first of the eight destination registers. (Extended, 64 bits).
```

Comment:
(Wow, I thought x86's PUSHA/POPA would always be an anomaly with 8 sources/dests...here we have AArch64 with just as extreme of operand counts.)